citesphere/src/main/java/edu/asu/diging/citesphere/web/user/SearchCitationsController.java

```diff
@@ -0,0 +1,31 @@
+package edu.asu.diging.citesphere.web.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.citesphere.core.search.service.SearchEngine;
+import edu.asu.diging.citesphere.core.search.service.impl.ResultPage;
+
+@Controller
+@PropertySource("classpath:/config.properties")
+public class SearchCitationsController {
+
+    @Autowired
+    private SearchEngine engine;
+
+    @RequestMapping(value = { "/auth/group/{zoteroGroupId}/references" })
+    public ResponseEntity<ResultPage> searchReferences(Authentication authentication, @PathVariable String zoteroGroupId,
```

Comment:
similar to the class name, this should not be called 'searchReferences' since it's not searching references, and the url should similarly be reflecting that it searches citations. Right now the url suggests it returns references.